.github/workflows/backport.yml

```diff
@@ -0,0 +1,45 @@
+# Derived from https://github.com/NixOS/nixpkgs/blob/2ab6f6d61630889491f86396b27a76ffb6fbc7bb/.github/workflows/backport.yml
+
+name: Backport
+
+on:
+  pull_request_target:
+    types: [closed, labeled]
+
+permissions: {}
```

Comment:
Despite explicitly disabling permissions with [`permissions: {}`](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token#defining-access-for-the-github_token-permissions), did you test this GitHub Action in a temporary repository to make sure that this repository (https://github.com/danth/stylix) has the necessary permissions for this Action to work? Otherwise, we could just merge this PR and validate it (in production) with a PR right after merging.

Reply:
The workflow gets its permissions via the GitHub App token which it creates in the first step, which is why the default token doesn't need any permissions on it.

I haven't tested this, but it's set up in the same way as upstream.